docs/changelog/5_x.rst

```diff
@@ -173,6 +172,9 @@ manage "feature" branches:
    then merge the EdgeDB branch as well (or rename the "feature" EdgeDB branch
    to "main", if the old branch is no longer needed).
 
+We've added ``edgedb branch`` commands to our CLI as well that create, copy,
+rename, drop, and rebase EdgeDB branches.
```

Comment:
```suggestion
We've added :ref:`edgedb branch commands <ref_cli_edgedb_branch>` to our CLI as
well that create, copy, rename, drop, and rebase EdgeDB branches.
```
In making this suggestion, I've noticed the visual links on the target page are wrong, so I'll fix that in another PR. (I had updated the hidden TOC before merging but not the visible links. 😞)

Reply:
Here's that PR to fix the branches command links: https://github.com/edgedb/edgedb/pull/7092